gcc/rust/util/rust-mapping-common.h

```diff
@@ -50,6 +50,14 @@ struct DefId
     return ((uint64_t) this->crateNum << 32 | this->localDefId)
 	   < ((uint64_t) other.crateNum << 32 | other.localDefId);
   }
+
+  std::string as_string () const
+  {
+    std::string buf;
+    buf += std::to_string (crateNum);
+    buf += std::to_string (localDefId);
```

Comment:
I think we might want to add a separator here so that the numbers aren't right next to each other?
```suggestion
    buf += std::to_string (crateNum);
    buf += " "; // or anything else
    buf += std::to_string (localDefId);
```